src/analytics/ledger/base_token.rs

```diff
@@ -27,7 +26,7 @@ pub struct BaseTokenActivityAnalytics {
 impl TransactionAnalytics for BaseTokenActivityAnalytics {
     type Measurement = BaseTokenActivity;
 
-    fn begin_milestone(&mut self, _: MilestoneIndex) {
+    fn begin_milestone(&mut self, _: MilestoneIndexTimestamp) {
         self.measurement = BaseTokenActivity::default();
     }
 
```

Comment:
Shouldn't we only count transferred value if the input doesn't have the same address as the outputs?

Reply:
you are right, this implementation looks wrong :thinking: 

for comparison: https://github.com/iotaledger/inx-analytics/blob/develop/pkg/analytics/ledger_output_analytics.go#L250-L264

Reply:
Thanks for pointing this out! Should be fixed now.

Reply:
@grtlr Please add comments. :trollface: